corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -215,6 +216,34 @@ public int printTableInfo(String namespace, String tablename) {
         return tableSize;
     }
 
+    /**
+     * Helper to analyze all the protobufs used in this cluster
+     */
+    public int printAllProtoDescriptors() {
+        int numProtoFiles = -1;
```

Comment:
actually as @zfrenette had pointed out returning 0, can hide a case where there should be some data but the test passes since 0 is the default value. Since this is for testing I would like to leave it at -1. please let me know if that's ok